src/js/components/Select/__tests__/__snapshots__/SelectMultiple-test.js.snap

```diff
@@ -1616,6 +1618,7 @@ exports[`Select Controlled multiple onChange with valueKey reduce 2`] = `
         aria-selected="false"
         aria-setsize="2"
         class="c5 c6"
+        option="[object Object]"
```

Comment:
Should this be coming through to the DOM?

Reply:
Yes this change is happening because styled components upgraded their version of `is-prop-valid` from v0.8.8 to v1.1.0 and in v1.1.0 `option` got added to the list of valid props. Here is the changelog: https://github.com/emotion-js/emotion/blob/main/packages/is-prop-valid/CHANGELOG.md#110

Reply:
Removed `option={option}` in latest commit